python/notebooks/05-using-the-planner.ipynb

```diff
@@ -513,7 +513,7 @@
     "        description=\"The search query\",\n",
     "    )\n",
     "    async def search_async(self, query: str, context: SKContext) -> str:\n",
-    "        query = query or context.variables.get(\"query\")[1]\n",
+    "        query = query or context.variables.get(\"query\")\n",
```

Comment:
the notebook runs with **0.3.12.dev0** where get still works the old way. Doesn't this break the notebook?

Reply:
good call, didn't notice the hardcoded version, is there a plan for how and when the notebooks upgrade version?